src/main/java/net/minecraftforge/event/AddReloadListenerEvent.java

```diff
@@ -66,6 +69,15 @@ public ICondition.IContext getConditionContext()
         return serverResources.getConditionContext();
     }
 
+    /**
+     * Provides access to the loaded registries associated with this reload.
+     * @return The RegistryAccess context for the currently active reload.
```

Comment:
Is this documentation correct? Given that no dynamically loaded content can be changed by a `/reload`, isn't this data the exact same before and after? Therefore, it should theoretically be possible to grab the registry access from ServerLifecycleHooks in the running server; however, it does probably make sense to add the access explicitly to this event.

Reply:
I'm going to go out on a limp and say that's /reload not working for certain data is a bug as shown in Jira.
https://bugs.mojang.com/browse/MC-260452
https://bugs.mojang.com/browse/MC-187938
https://bugs.mojang.com/browse/MC-260446

Would make sense to future proof for when these bugs do get fix and we do need the right registryAccess passed in

Reply:
I'm not 100% sure if the registryaccess is accessible via the server on the initial reloadlistener load (especially for singleplayer create world)

Reply:
The registry data will stay unchanged after `/reload`, but it'll at least be accessible throughout - but in retrospect, this is quite ambiguous. 😄 Will try something else 🙂 

And yep, exactly - the server is not always created by this point.

Reply:
Tweaked a bit, although I'm still unsure. Could specifically clarify the case with registries not being `/reload`-able, but it of course has potential to become stale in the future. 🙂